proto/proto-backplane-grpc/src/main/proto/deephaven/proto/table.proto

```diff
@@ -356,6 +368,33 @@ message UpdateByEmaTimescale {
   }
 }
 
+// Reusable options for the UpdateBy exponential moving operations.
+message UpdateByEmaOptions {
+  // If unset, defaults to server-provided defaults.
+  optional BadDataBehavior on_null_value = 1;
```

Comment:
message fields are implicitly optional, you don't need to specify this i believe?